dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -1271,14 +1272,17 @@ BlockInputStreams DeltaMergeStore::read(
     LOG_INFO(
         tracing_logger,
         "Read create stream done, keep_order={} dt_enable_read_thread={} enable_read_thread={} "
-        "is_fast_scan={} is_push_down_filter_empty={} pool_id={} num_streams={}",
+        "is_fast_scan={} is_push_down_filter_empty={} pool_id={} num_streams={} columns_to_read={} "
+        "final_columns_to_read={}",
         keep_order,
         db_context.getSettingsRef().dt_enable_read_thread,
         enable_read_thread,
         is_fast_scan,
         filter == nullptr || filter->before_where == nullptr,
         read_task_pool->pool_id,
-        final_num_stream);
+        final_num_stream,
+        columns_to_read,
+        final_columns_to_read);
```

Comment:
Will this output the column_name to the logging? Can we only output the column_id?